config/database.yml

```diff
@@ -24,8 +24,8 @@ default: &default
 development:
   <<: *default
   database: proyecto_final_pf_f22_t3_development
-  username: 
-  password: 
+  username: <%= Rails.application.credentials.database.username %>
+  password: <%= Rails.application.credentials.database.password %>
 
   # The specified database role being used to connect to postgres.
   # To create additional roles in postgres see `$ createuser --help`.
```

Comment:
This file should not appear on GitHub, instead there should be an example file.

Reply:
Ok, removed file thx!